packages/monocdk-experiment/gen.js

```diff
@@ -0,0 +1,99 @@
+// generates the mono-cdk module by copying @aws-cdk/lib/** to src/
+// and rewriting the interdepedent "import" statements.
+const fs = require('fs-extra');
+const path = require('path');
+const glob = require('glob');
+
+async function main() {
+  const srcdir = path.resolve('src');
```

Comment:
Maybe tighten this up a little more... Right now it appears very dependent on the current working directory... And you'd remove `src` under there... Dangerous?

```suggestion
  const srcdir = path.resolve(__dirname, 'src');
```

Reply:
That's intentional. This is just a tool that produces `src`.